### DISCUSSION TITLE
LFRic Apps Release

### DISCUSSION THREAD

**Discussion by jennyhickson** [MetOffice/simulation-systems#163] [Announcements]
Hello,

**We are very pleased to announce that the LFRic repository split has been successfully achieved and vn1.0 of [LFRic Apps](https://code.metoffice.gov.uk/trac/lfric_apps/wiki) is now ready for use. LFRic Core is also now open for reviews and changes again.** 

Thanks to everyone who worked to make this a success – both in developing and reviewing tickets. 

There is guidance on how to migrate branches from the LFRic Core repository to the LFRic Apps repository here [lfric_apps/wiki/MigratingLFRicBranches](https://code.metoffice.gov.uk/trac/lfric_apps/wiki/MigratingLFRicBranches) and if you’d like further assistance we are running two drop in sessions on **16th April 14:00** and **24th April 09:30** where we’ll be happy to discuss any questions you might have. 

The LFRic Apps repository is being managed using the [Simulation Systems Working Practices](https://metoffice.github.io/simulation-systems/), and we’d encourage you to take a look through these before you start. Of notable difference from the LFRic Core repository:
•	We are asking for branches from the last released version wherever possible. To create a new apps branch you would therefore use fcm:lfric_apps.x_tr@vn1.0. 
•	Please do not merge the head into your working branch – if you need to bring your work up to the latest revisions then create a head of trunk branch and merge your changes onto that. 
•	Code reviewers should be requested by emailing [umsysteam@metoffice.gov.uk](mailto:umsysteam@metoffice.gov.uk) and these will be distributed between the SSD and CCD teams. 
•	LFRic Apps will be released regularly alongside the UM, JULES and UKCA. Changes wishing to make any given release should be with their code reviewer before the CR deadline. The deadline for vn1.1 is 31st May 2024 and we will remind you of this closer to the time. 

Thanks again for all your help and patience during this change and do get in touch with any questions. 

Jenny, on behalf of the SSD Team

**Comment by richardgilham** [comment]
Thank you and well done to everyone that contributed towards this massive achievement. This was a very complex and high stakes change that has gone incredibly well.
